frontend/src/components/PlacementRuleDeprecationAlert.tsx

```diff
@@ -2,9 +2,9 @@
 
 import { Alert } from '@patternfly/react-core'
 import React from 'react'
-import { useTranslation } from '../../../../../../lib/acm-i18next'
+import { useTranslation } from '../lib/acm-i18next'
 
-const DeprecationAlert = () => {
+const PlacementRuleDeprecationAlert = () => {
```

Comment:
Optional: This could stay `DeprecationAlert` and take current and replacement props to make it reusable.